website/src/components/TOS/SectionCard.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, Flex, Heading, Stack, Text, useColorModeValue } from "@chakra-ui/react";
+
+import { ChapterInfo, SectionInfo } from "./PolicyContents";
+
+export interface ChapterProps {
+  chapter: ChapterInfo;
+}
+
+export interface SectionProps {
+  section: SectionInfo;
+}
+
+export const ChapterCard = ({ chapter }: ChapterProps) => {
+  const backgroundColor = useColorModeValue("gray.100", "gray.800");
+
+  return (
+    <Box bg={backgroundColor} p="6" borderRadius="xl" shadow="base">
+      <Stack spacing="4">
+        <Stack>
+          <Flex alignItems="end" gap="2">
+            <Text as="b" fontSize="xl" color="blue.500">
+              {chapter.number}
+            </Text>
+            <Heading as="h3" size="lg">
+              {chapter.title}
+            </Heading>
+          </Flex>
+          <Text>{chapter.desc}</Text>
+        </Stack>
+        {chapter.sections && chapter.sections.length
+          ? chapter.sections.map((section, sectionIndex) => <SectionCard key={sectionIndex} section={section} />)
+          : ""}
+      </Stack>
+    </Box>
+  );
+};
+
+export const SectionCard = ({ section }: SectionProps) => {
```

Comment:
Rather than putting both of these components in the same file, can we split them up into two separate files:
 - `website/src/components/PolicySectionCard/PolicySectionCard.tsx`
 - `website/src/components/PolicyChapterCard/PolicyChapterCard.tsx`

Then, later on it'll be easy to add the Storybook stories associated with these and make UI revisions easier.  Once the MVP is settled, I'll be pushing us to break up all the components into simpler sub directories like these.

Reply:
Also by splitting these up you can do `export default const ...` and import via `import PolicySectionCard from ...`